cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +77,25 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
+  OUTPUT_NAME "igfxcmrt"
```

Comment:
isn't it the default? you can try to drop OUTPUT_NAME setting completely - it should work I think